### PR TITLE
Moonpile/issue84

### PR DESCRIPTION
**Pull request by Moonpile**
Highlights weapons where the character's STR or DEX does not meet the requirements.  Note that this does not enforce the penalty on any rolls.

### CODE REVIEW

File: src/actors/rqgActorSheet.ts [hunk 1]
```diff
@@ -417,6 +418,50 @@ export class RqgActorSheet extends ActorSheet<
       (a: any, b: any) => b.data.data.dieFrom - a.data.data.dieFrom
     );
 
+    //@ts-ignore
+    itemTypes[ItemTypeEnum.Weapon].forEach(weapon => {
+      assertItemType(weapon.data.type, ItemTypeEnum.Weapon);
+      console.log(weapon);
```
Comment (wake42) [inline src/actors/rqgActorSheet.ts:424]: remove console log

File: src/actors/rqgActorSheet.ts [hunk 2]
```diff
@@ -417,6 +418,50 @@ export class RqgActorSheet extends ActorSheet<
...
+      let usages = weapon.data.data.usage;
+      let actorStr = this.actor.data.data.characteristics.strength.value;
+      let actorDex = this.actor.data.data.characteristics.dexterity.value;
+      for (const key in usages) {
+        let usage = usages[key];
+        if (usage.skillId) {
+          console.log("KEY");
+          console.log(key);
+          console.log("Value");
+          console.log(usages[key]);
```
Comment (wake42) [inline src/actors/rqgActorSheet.ts:435]: remove console logs

**Review by wake42 — CHANGES_REQUESTED** [review]

File: src/actors/rqgActorSheet.ts (hunk 1 above)
Comment (Moonpile) [reply src/actors/rqgActorSheet.ts:424]: Done

File: src/actors/rqgActorSheet.ts (hunk 2 above)
Comment (Moonpile) [reply src/actors/rqgActorSheet.ts]: done

**Review by wake42 — APPROVED** [review]

**Pull request merged**